tests/test_country_codes_standalone.py

```
@@ -0,0 +1,51 @@
+def test_get_country_name_known_codes():
+    for code, expected in CODE_TO_NAME.items():
+        assert geocoding.get_country_name(code) == expected
+
+
+def test_get_country_code_known_names():
+    for name, expected in {v: k for k, v in CODE_TO_NAME.items()}.items():
+        assert geocoding.get_country_code(name) == expected
+
+
+def test_round_trip():
+    for code, name in CODE_TO_NAME.items():
```

Ruff (0.15.12)

[warning] Loop control variable `name` not used within loop body

Rename unused `name` to `_name`

(B007)